README.md

```diff
@@ -1,8 +1,8 @@
 # Batch Learning Forecasting Component
 
-The component enables using external predictive models from  [Scikit Learn](http://scikit-learn.org/stable/index.html) library (for example [Random Forest Regressor](http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html)) implementation in a streaming scenario. Fitting, saving, loading and live prediction are enabled. Live predictions work via Kafka streams (reading feature vectors from Kafka and writing predictions to Kafka).
+The component enables using external predictive models from  [Scikit Learn](http://scikit-learn.org/stable/index.html) and [PyTorch](https://pytorch.org/) library (for example [Random Forest Regressor](http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html)) implementation in a streaming scenario. Fitting, saving, loading and live prediction are enabled. Live predictions work via Kafka streams (reading feature vectors from Kafka and writing predictions to Kafka).
```

Comment:
Please fix the order. Randm Forest Regressor should be adjacent to Scikit Learn.